### PR TITLE
feat: extend kubernetes perf tuning section

### PR DESCRIPTION
**Pull request by Automaat**
In 2.3.x version we have added a couple of configuration option that could help with performance issues on big k8s cluster

Did you sign your commit? [Instructions](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md#sign-your-commits)

Have you read [Contributing guidelines](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Review by lahabana — APPROVED** [review]

File: app/_src/production/upgrades-tuning/fine-tuning.md [hunk 1]
```diff
@@ -95,14 +95,48 @@ After a successful debugging session, please remember to turn off the debugging
...
 
 You can enable this only after all instances of the control plane are updated to 1.6.0 or later.
 This option will be the default behaviour in the next versions of {{site.mesh_product_name}}.
 {% endif_version %}
+
+{% if_version gte:2.3.x %}
+
+### Kubernetes client
+
+Kubernetes client uses client level throttling to not overwhelm kube-api server. In larger deployments, bigger than 2k services in a single kubernetes cluster, number
```
Comment (slonka) [inline app/_src/production/upgrades-tuning/fine-tuning.md:114]:
```suggestion
Kubernetes client uses client level throttling to not overwhelm kube-api server. In larger deployments, bigger than 2000 services in a single kubernetes cluster, number
```

File: app/_src/production/upgrades-tuning/fine-tuning.md [hunk 2]
```diff
@@ -95,14 +95,48 @@ After a successful debugging session, please remember to turn off the debugging
...
+runtime:
+  kubernetes:
+    clientConfig:
+      qps: ... # Qps defines maximum requests kubernetes client is allowed to make per second.
+      burstQps: ... # BurstQps defines maximum burst requests kubernetes client is allowed to make per second
+```
+
+### Kubernetes controller manager
+
+{{site.mesh_product_name}} is modifying some kubernetes resources. Kubernetes call the process of modification reconciliation. Every resource has its own working queue, and control plane adds reconciliation tasks to that queue. In larger deployments, bigger than 2k services in a single kubernetes cluster, size of the work queue for pod reconciliation
```
Comment (slonka) [inline app/_src/production/upgrades-tuning/fine-tuning.md:128]:
```suggestion
{{site.mesh_product_name}} is modifying some Kubernetes resources. Kubernetes calls the process of modification reconciliation. Every resource has its own working queue, and control plane adds reconciliation tasks to that queue. In larger deployments, bigger than 2000 services in a single Kubernetes cluster, size of the work queue for pod reconciliation
```

File: app/_src/production/upgrades-tuning/fine-tuning.md [hunk 3]
```diff
@@ -95,14 +95,48 @@ After a successful debugging session, please remember to turn off the debugging
...
+can grow and slow down pods updates. In this situation you can change the number of concurrent pod roconciliation tasks, by changing configuration:
```
Comment (slonka) [inline app/_src/production/upgrades-tuning/fine-tuning.md:129]:
```suggestion
can grow and slow down pods updates. In this situation you can change the number of concurrent pod reconciliation tasks, by changing configuration:
```

**Review by slonka — APPROVED** [review]
just small stuff

**Comment by slonka** [comment]
btw, how come typos were not caught by vale?

**Pull request merged**